docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -34,7 +35,15 @@ Fixes #
 ## My Calva PR Checklist
 <!-- Strike out (using `~`) items that do not apply, as Github reports how many are not ticked. If you want to add checkboxes, please do. -->
 
-I have:
+If this PR involves only documentation changes, I have:
+
+- [ ] Read [Editing Documentation](https://github.com/BetterThanTomorrow/calva/wiki/How-to-Hack-on-Calva#editing-documentation)
+- [ ] Directed this pull request at the `published` branch.
+- [ ] Referenced the issue I am fixing/addressing _in a commit message for the pull request_ (if there was is an issue for the documentation change)
+  - [ ] If I am fixing the issue, I have used [GitHub's fixes/closes syntax](https://help.github.com/en/articles/closing-issues-using-keywords)
+  - [ ] If I am fixing just part of the issue, I have just referenced it w/o any of the "fixes” keywords.
```

Comment:
I think a check like *Built the docs site and tested it locally*. should be added here. Probably guarded with that this is not needed for simple typo fixes.